libs/sdl/sdl.c

```diff
@@ -80,9 +93,15 @@ HL_PRIM bool HL_NAME(init_once)() {
 	timeBeginPeriod(1);
 #	endif
 	// default GL parameters
+#if TARGET_OS_IOS || TARGET_OS_TV || __ANDROID__

```

Comment:
please replace such checks with defined(HL_IOS) || defined(HL_ANDROID)
or maybe if there many of them you want to have  #ifdef HL_MOBILE ?

Reply:
Yes, fixed in last commit with an added HL_MOBILE #define symbol.